app/javascript/gobierto_dashboards/modules/costs_controller.js

```diff
@@ -0,0 +1,177 @@
+import Vue from "vue";
+import VueRouter from "vue-router";
+
+import { getRemoteData } from '../webapp/lib/utils'
+
+Vue.use(VueRouter);
+Vue.config.productionTip = false;
+
+export class CostsController {
+  constructor(options) {
+    this.charts = {};
+
+    const selector = "gobierto-dashboards-costs-app";
+
+    // Mount Vue applications
+    const entryPoint = document.getElementById(selector);
+
+    if (entryPoint) {
+      const htmlRouterBlock = `<router-view></router-view>`;
+
+      entryPoint.innerHTML = htmlRouterBlock;
+
+      const Home = () => import("../webapp/containers/costs/Home.vue");
+      const TableFirstLevel = () => import("../webapp/containers/costs/table/TableFirstLevel.vue");
+      const TableSecondLevel = () => import("../webapp/containers/costs/table/TableSecondLevel.vue");
+      const TableItem = () => import("../webapp/containers/costs/table/TableItem.vue");
+
+      Promise.resolve(getRemoteData(options.costsEndpoint)).then((rawData) => {
+        this.setGlobalVariables(rawData)
+
+        const router = new VueRouter({
+          mode: "history",
+          routes: [
+            {
+              path: "/dashboards/costes/",
+              name: 'Home',
+              component: Home,
+              children: [
+              {
+                path: "/dashboards/costes/:year?",
+                component: TableFirstLevel,
+                name: 'TableFirstLevel'
+              },
+              {
+                path: "/dashboards/costes/:year?/:id?",
+                component: TableSecondLevel,
+                name: 'TableSecondLevel'
+              },
+              {
+                path: "/dashboards/costes/:year?/:id?/:item?",
+                component: TableItem,
+                name: 'TableItem'
+              }
+              ]
+            },
+          ],
+          scrollBehavior(to) {
+            let element
+            //Get a different position scroll
+            if (to.name !== 'TableFirstLevel') {
+              element = document.getElementById('gobierto-dashboards-title-detail');
+              window.scrollTo({ top: element.offsetTop, behavior: "smooth" });
+            }
+          }
+        });
+
+        const baseTitle = document.title;
+        router.afterEach(to => {
+          // Wait 2 ticks
+          Vue.nextTick(() =>
+            Vue.nextTick(() => {
+              let title = baseTitle;
+
+              if (to.name === "costs") {
+                const { item: { title: itemTitle } = {} } = to.params;
+
+                if (itemTitle) {
+                  title = `${itemTitle}${baseTitle}`;
```

Comment:
This is not working and would be great to update the title of the page with the item your are watching